packages/rule-components/src/ReportDetails/ReportDetails.css

```diff
@@ -0,0 +1,12 @@
+.OCMReportDetails {
+    padding: 20px 40px;
+}
+
+.OCMReportDetailsExternalLinkIcon {
```

Comment:
`ins-c-rule__report-detail-link`

Also this should be child style of `ins-c-rule__report-detail`